learn/what_is_meilisearch/comparison_to_alternatives.mdx

```diff
@@ -106,12 +107,25 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 | Distinct attributes <br /><div style={{color: "#A9A9A9", fontSize: "0.9em"}}>De-duplicate documents by a field value</div>| ✅ | ✅ | ✅  | ✅ |
 | Grouping <br /><div style={{color: "#A9A9A9", fontSize: "0.9em"}}>Bucket documents by field values</div> | ❌ | ✅ | ✅  | ✅ |
 
+##### AI Search
+
+|   | Meilisearch | Algolia | Typesense | Elasticsearch |
+|---|:---:|:----:|:---:|:---:|
+| Semantic Search | ✅ | 🔶 <br /> Under Premium plan | ✅ | ✅ |
+| Hybrid Search | ✅ | 🔶 <br /> Under Premium plan | ✅ | ✅ |
+| Embedding Generation | ✅  <br /> OpenAI <br /> HuggingFace <br /> | Undisclosed | <br/> OpenAI <be /> Google PaLM <br /> GCP Vertex AI | ❌  |
+| Prompt Embedding | ✅ | Undisclosed | ❌ | ❌ | 
+| Vector Store | ✅ | Undisclosed | ✅ | ✅ |
+| Langchain Integration | ✅ | ❌ | ✅ | ✅ |
+| GPU support | ✅ <br /> CUDA | Undisclosed | ✅ <br /> CUDA | ❌ |
+
 ##### Visualize
 
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | [Mini Dashboard](https://github.com/meilisearch/mini-dashboard) | ✅ | 🔶 <br /> Cloud product | 🔶 <br /> Cloud product | ✅ |
 | Search Analytics | ✅ <br /> [Cloud product](https://www.meilisearch.com/cloud) | ✅ <br /> Cloud Product | ❌ | ✅ <br /> Cloud Product |
+| Monitoring Dashboard | ✅ <br /> [Cloud product](https://www.meilisearch.com/cloud) | ✅ <br /> Cloud Product | ✅ <br /> Cloud Product | ✅ <br /> Cloud Product |
```

Comment:
Is there a reason why we'd refer to it as monitoring dashboard instead of monitoring metrics?

Reply:
_Metrics_ may be colliding with _Analytics_ (that's the only reason I got in mind); We got a past discussion internally about renaming both features to have something clearer but we can go with Monitoring Metrics right now